backend/app/controllers/memos_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class MemosController < ApplicationController
+  def create
```

Comment:
IMO

以下のように、エンドポイントのコメントがあると親切かなと思いました〜！
（routes.rbのぱっと見だと、エンドポイントが少し分かりづらいのであった方が親切だと思っている。）
```suggestion
class MemosController < ApplicationController
  # POST /memos
  def create
```